test/unit/modules/peers.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
```

Comment:
There several things in this test file:
- Change `should insert new peer with only ip and port and state defined` to `should insert new peer with only ip, port and state`
- `should not insert new peer if address changed but nonce is the same` is duplicated
-  I propose to create a context block onto the `list` block state to state we are inserting a peer. In that context we could create a beforeEch with the line `peers.update(randomPeer);`
-